src/ShiftedProximalOperators.jl

```diff
@@ -52,6 +55,7 @@ include("shiftedCappedl1.jl")
 include("shiftedNuclearnorm.jl")
 
 (ψ::ShiftedProximableFunction)(y) = ψ.h(ψ.xk + ψ.sj + y)
+(ψ::ShiftedCompositeProximableFunction)(y) = (!ψ.is_shifted) ? ( z = similar(ψ.b);ψ.c!(y,z);ψ.h(z)) : ψ.h(ψ.b + ψ.A*y)
```

Comment:
```suggestion
(ψ::ShiftedCompositeProximableFunction)(y) = begin
  if ψ.is_shifted
    return ψ.h(ψ.b + ψ.A * y)
  end
  z = similar(ψ.b)
  ψ.c!(y, z)
  ψ.h(z))
end
```

Reply:
In Julia, the convention is that the argument that is modified by a `!` function appears first in the argument list. When you write `c!(y, z)`, I presume you mean that `z = c(y)`. We would normally write it `c!(z, y)`.